### PR TITLE
GH-950: Create Sermon Notes Manager

### PR DESCRIPTION
**Pull request by JoshNg123**

### CODE REVIEW

File: ui/src/components/admin/sermonNotes/AdminSermonNotesContainer.js [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+        Sermon Notes Manager
+      </Heading>
+      <Stack direction="row" mb={5}>
+        <Button
+          colorScheme="blue"
+          variant={isEditorOpen ? 'solid' : 'outline'}
+          borderColor="#3182CE"
+          borderWidth={3}
+          size="lg"
+          disabled={isPublishDisabled}
```
Comment (Baeck016) [inline ui/src/components/admin/sermonNotes/AdminSermonNotesContainer.js:232]: this is doing reference to the function. But you want to call the function so needs bracket

File: ui/src/components/admin/sermonNotes/AdminSermonNotesContainer.js [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+          borderWidth={3}
+          size="lg"
+          onClick={() => setIsEditorOpen(false)}
+        >
+          Past Sermons
+        </Button>
+      </Stack>
+      {!isEditorOpen && (
+        <>
+          {/* <Stack direction={['row']} w="100%" spacing={5}>
```
Comment (Baeck016) [inline ui/src/components/admin/sermonNotes/AdminSermonNotesContainer.js:250]: can delete

**Comment by Baeck016** [comment]
스크린샷 2024-03-27 175640
The buttons are going out of the box

**Comment by Baeck016** [comment]
스크린샷 2024-03-27 180634
Can change button color because the word isn't that visible

**Comment by Midnite1212** [comment]
I think you rebased your branch wrongly here. There's a lot of commits that's not supposed to be here, like the be and the tiptap configs. You can try reverting the commits from be and tiptap, or you can create a fresh branch and cherry pick your commits. But since thers so many commits not meant to be here, better to create a new branch :")

**Comment by Midnite1212** [comment]
moved to another pr that has been merged

**Pull request closed without merging**